e2e-tests/tests/util.ts

```diff
@@ -42,33 +51,60 @@ export async function customRequest(web3: Web3, method: string, params: any[]) {
 		);
 	});
 }
+export interface ExtendedAssetHubApi extends ApiPromise {
+	laosAssetId: StagingXcmV3MultiLocation;
+	relayAssetId: StagingXcmV3MultiLocation;
+}
```

Comment:
Why do you think it may lead to confusion? They are types that belong to AssetHub api. I see the context more related to the networks configuration for the tests. Not sure that adding api Asset Hub specific types to it is a good idea. But open to hear more suggestions

Reply:
I see your point, maybe context isn't the best place for them either. I see the Api as an interface to send transactions and queries to the node more than to a place for storing values, that's why I think it might become confusing, but this is just a personal opinion, I agree that we have to store them somewhere.

Additionally, why those assetId are worthy to be included here and other AssetHub items do not?

I'd suggest to either create a data structure containing all AssetHub/LAOS items (assets ids from the parachain perspective, addresses, locations, ...) or to include those types in this ``ExtendedAssetHubApi`` interface, so we'll have them available in all our tests, I'm sure we'll need them in the future. What do you think?